### PR TITLE
Fixes magento/devdocs#6025

### LINKED ISSUE #6025: Advanced JS Bundling examples can lead to problems with JS execution
# General issue

## Description:

Using the current guide to setup JS bundling can lead to bundles which include `domReady!` in the bundle.

This results in the main bundle to be included multiple times on the page, which can cause problems with jQuery and other javascript components.

`domReady!` is included due to the manual writing of the bundle configuration, which doesn't filter out undesireable content (which `bundlesConfigOutFile` does for some reason).

## Possible solutions:

Using `bundlesConfigOutFile` doesn't exhibit this behavior.

## Additional information:

Example config

```
# build.js

modules: [
    {
        name: 'requirejs/require'
    },
    {
        name: 'mage/bootstrap',
        include: [
            ....
        ]
    }
]
``` 

```
# requirejs-config.js

require.config({
   bundles: {"requirejs/require":[...,"requirejs/domReady","domReady!",...]},
});require.config({
   bundles: {"mage/bootstrap": [....]},
});
....

```

DOM HTML, after JS has been allowed to run

```html
<head>
...
<script type="text/javascript" src="THEME_URL/da_DK/requirejs/require.js"></script>

...
<script type="text/javascript" charset="utf-8" async="" data-requirecontext="_" data-requiremodule="domReady!" src="THEME_URL/da_DK/requirejs/require.js"></script>
```

### PR DESCRIPTION
**Pull request by k4emic**
## Purpose of this pull request

Fixes #6025 

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/performance-best-practices/advanced-js-bundling.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/performance-best-practices/advanced-js-bundling.md [hunk 1]
```diff
@@ -479,11 +479,11 @@ Open `requirejs-config.js` in the `pub/static/frontend/Magento/luma/en_US` direc
...
-    bundles: { "bundles/default": ["mage/template","mage/apply/scripts","mage/apply/main","mage/mage","mage/translate","mage/loader"] }
-});
-require.config({
-    bundles: { "bundles/cart": ["Magento_Ui/js/lib/validation/utils","Magento_Ui/js/lib/validation/rules","Magento_Ui/js/lib/validation/validation"] }
-});
+    bundles: {
+        "bundles/default": ["mage/template", "mage/apply/scripts", "mage/apply/main", "mage/mage", "mage/translate", "mage/loader"],
+        "bundles/cart": ["Magento_Ui/js/lib/validation/utils", "Magento_Ui/js/lib/validation/rules", "Magento_Ui/js/lib/validation/validation"]
+    }
+}
```
Comment (k4emic) [inline guides/v2.3/performance-best-practices/advanced-js-bundling.md]: Bundles are now written in a single require.config instruction. This output might contain more spacing (one module per line) depending on whether you're using node.js, rhino or something else.

**Comment by k4emic** [comment]
Signed CLA

**Comment by dobooth** [comment]
running tests

**Pull request merged**